frontend/src/components/ApplicationReview/ApplicationOverview.tsx

```diff
@@ -33,31 +32,51 @@ export interface IUserInfoObject {
 export interface IRoomInfoObject {
   [_PARTNER]?: string;
   [_PREFERRED_ROOM]?: string;
-  [_SEAT_ROLLOVER]?: string;
+  [_SEAT_ROLLOVER]?: boolean;
 }
-const ApplicationOverview: React.FunctionComponent<IProps> = (props) => {
-  const { applicationInfo, title } = props;
-  const userInfoObject: IUserInfoObject = {
-    [_NAME]: applicationInfo.user !== undefined
-      ? applicationInfo.user.username
-      : applicationInfo.fullname,
-    [_EMAIL]: applicationInfo.email,
-    [_PHONE]: applicationInfo.phone,
-    [_MASTER_STATUS]: applicationInfo.status,
+
+export interface IInformationObject {
+  [_COMMENTS]?: string;
+  [_NEEDS]?: string;
+}
+
+const ApplicationOverview: React.FunctionComponent<IStateProps> = (props) => {
+  const { applicationInfo } = props;
+  const { partnerApplication } = applicationInfo;
+  const userInfoObject: IUserInfoObject = applicationInfo.user !== undefined
+  ?
+  {
+    [_NAME]: applicationInfo.user.username,
+    [_EMAIL]: applicationInfo.user.email,
+    [_PHONE]: undefined,
+    [_MASTER_STATUS]: applicationInfo.user.masterStatus,
+  }
+  :
+  {
+    [_NAME]: undefined,
+    [_EMAIL]: undefined,
+    [_PHONE]: undefined,
+    [_MASTER_STATUS]: undefined,
   };
   const roomInfoObject: IRoomInfoObject = {
-    [_PARTNER]: applicationInfo.partner,
-    [_PREFERRED_ROOM]: applicationInfo.room,
+    [_PARTNER]: partnerApplication !== undefined
+      && partnerApplication.user !== undefined
+      ? partnerApplication.user.username
+      : '',
+    [_PREFERRED_ROOM]: applicationInfo.preferredRoom,
     [_SEAT_ROLLOVER]: applicationInfo.seatRollover,
   };
+  const informationInfoObject: IInformationObject = {
+    [_COMMENTS]: applicationInfo.comments,
+    [_NEEDS]: applicationInfo.needs,
+  };
+
   return (
     <div id="application-review">
-      <Sidetittel>{title ? title : _REVIEW_APPLICATION}</Sidetittel>
```

Comment:
This locks the title to always be _REVIEW_APPLICATION, and it is used elsewhere where the title doesn't fit